cernopendata/jsonschemas/records/glossary-term-v1.0.0.json

```diff
@@ -1,6 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
-	"description": "Describe information needed for term entity.",
+	"description": "Describe information needed for glossary term entity.",
```

Comment:
Perhaps we could have simply used `glossary-v1.0.0.json` everywhere, in the schema and templates etc.... for the sake of simplicity.